opentelemetry-otlp/src/logs.rs

```diff
@@ -108,7 +108,16 @@ impl HasHttpConfig for LogExporterBuilder<HttpExporterBuilderSet> {
 /// OTLP exporter that sends log data
 #[derive(Debug)]
 pub struct LogExporter {
-    client: Box<dyn opentelemetry_sdk::export::logs::LogExporter>,
+    //client: Box<dyn opentelemetry_sdk::export::logs::LogExporter>,
+    client: LogExporterInner,
+}
+
+#[derive(Debug)]
+enum LogExporterInner {
```

Comment:
The name can be better IMO. Maybe `SupportedTransportClient`. `Inner` just don't convey this is a enum clearly

Reply:
Good point. Have renamed it to `SupportedTransportClient`.